windows/web/slimevr_web_installer.nsi

```diff
@@ -378,6 +378,43 @@ Section "SlimeVR Server" SEC_SERVER
     WriteUninstaller "$INSTDIR\uninstall.exe"
 SectionEnd
 
+Section "Webview2" SEC_WEBVIEW
+    SectionIn RO
+    Var /GLOBAL webviewKeyA
+    Var /GLOBAL webviewKeyB
+
+
+    # detecting webview from the windows registry
+    ${If} ${RunningX64}
+        ReadRegStr $webviewKeyA HKLM "SOFTWARE\WOW6432Node\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}" "pv"
+        ReadRegStr $webviewKeyB HKCU "Software\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}" "pv"
+    ${Else}
+        ReadRegStr $webviewKeyA HKLM "SOFTWARE\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}" "pv"
+        ReadRegStr $webviewKeyB HKCU "Software\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}" "pv"
+    ${EndIf}
+
+    ${If} $webviewKeyA == 0 
+    ${AndIf} $webviewKeyB == 0
```

Comment:
You're comparing str to int, this will always be false. It should be `${If} $webviewKeyA == ""` in case you want to check whether the key is not present.